src/test/kotlin/de/jodamob/kotlin/testrunner/sample/TestWithRunner.kt

```diff
@@ -1,15 +1,17 @@
 package de.jodamob.kotlin.testrunner.sample
 
 import de.jodamob.kotlin.testrunner.KotlinTestRunner
+import de.jodamob.kotlin.testrunner.OpenedClasses
+import de.jodamob.kotlin.testrunner.nonopen.FinalNonOpenClassSample
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito
 import org.mockito.exceptions.base.MockitoException
 
 @RunWith(KotlinTestRunner::class)
+@OpenedClasses("de.jodamob.kotlin.testrunner.sample")
```

Comment:
how about having 2 annotations: one for package, one for classes so its more obvious
and instead of having this forces to set, the default would grab the package from the testclass